src/java.base/share/classes/java/lang/invoke/MethodType.java

```diff
@@ -394,20 +393,11 @@ private static MethodType makeImpl(Class<?> rtype, Class<?>[] ptypes, boolean tr
             ptypes = NO_PTYPES; trusted = true;
         }
         MethodType primordialMT = new MethodType(rtype, ptypes);
-        if (archivedMethodTypes != null) {
-            // If this JVM process reads from archivedMethodTypes, it never
-            // modifies the table. So there's no need for synchronization.
-            // See copyInternTable() below.
-            assert CDS.isUsingArchive();
-            MethodType mt = archivedMethodTypes.get(primordialMT);
-            if (mt != null) {
-                return mt;
-            }
-        }
-
         MethodType mt = internTable.get(primordialMT);
         if (mt != null)
+        {
```

Comment:
This left brace does not follow the code style in this file. Either remove this pair of brace or make it `if (mt != null) {`